massa-execution-worker/src/tests/scenarios_mandatories.rs

```diff
@@ -98,6 +99,91 @@ fn test_sending_read_only_execution_command() {
     manager.stop()
 }
 
+/// Test the gas usage in nested calls using call sc operation
+///
+/// Create a smart contract and send it in the blockclique.
+/// This smart contract have his sources in the sources folder.
+/// It calls the test function that have a sub-call to the receive function and send it to the blockclique.
+/// We are checking that the gas is going down through the execution even in sub-calls.
+///
+/// This test can fail if the gas is going up in the execution
```

Comment:
Maybe add something explaining that gas going up probably means that sub calls subs were not taken in account.

Reply:
Ok maybe not.